src/main/java/com/example/Game.java

```diff
@@ -10,44 +10,36 @@ class Game {
     private final Scanner scanner = new Scanner(System.in);
 
     public Game() {
-        System.out.println("Bored!Let's play a Game!!\uD83C\uDFB2");
-        System.out.println("Enter the grid size: (3 for 3x3, 4 for 4x4 etc)");
+        System.out.println("Bored! Let's play a Game!! \uD83C\uDFB2");
+        System.out.println("Enter the grid size: (3 for 3x3, 4 for 4x4, etc)");
         int size;
-        while (true)
-        {
-            try{
+        while (true) {
```

Comment:
Do Not interact with user in constructor of class,
use different methods to do that,
refer the comment made in first review.

Reply:
Done